domains/mh.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "developermehedihasan",
+        "email": "bslamraopari@gmail.com"
+    },
+    "record": {
+        "A": ["185.199.108.153"]
```

Comment:
I cannot reach your website (404)

if you want to use github pages, it uses 4 IPs
- `185.199.108.153`
- `185.199.109.153`
- `185.199.110.153`
- `185.199.111.153`